OCPP-J/src/main/java/eu/chargetime/ocpp/WebSocketListener.java

```diff
@@ -122,19 +124,25 @@ public ServerHandshakeBuilder onWebsocketHandshakeReceivedAsServer(WebSocket web
                             .InternetAddress(webSocket.getRemoteSocketAddress())
                             .build();
 
-            String username = null, password = null;
+            byte[] username = null, password = null;
```

Comment:
The username is the charge point identity, which is a printable string, so I'd pass that as a String.

Reply:
Changed to String. Please review.